assets/app/styles/_spacers.less

```diff
@@ -0,0 +1,36 @@
+// Spacing utility classes
+// creates:
+//  .pad-sm, .pad-md, .pad-lg, etc
+//  .mar-sm, .mar-md, .mar-lg, etc
+//  .pad-left-sm, .mar-left-sm, etc
+//
+@sizes: none 0px e('!important'), sm 4px e(''), md 8px e(''), lg 12px e(''), xl 16px e(''), xxl 32px e('');
```

Comment:
We tend to use multiples of 5px today. Not that we have to stick to that.

A quick check with some fuzzy, probably incorrect regex.

```
assets❯ ag --less --html '(padding|margin).*:.* 5px' | wc -l                                                                                                                                                                                     origin/git/debug-terminal-1.3
      55
assets❯ ag --less --html '(padding|margin).*:.* 10px' | wc -l                                                                                                                                                                                    origin/git/debug-terminal-1.3
      74
assets❯ ag --less --html '(padding|margin).*:.* 15px' | wc -l                                                                                                                                                                                    origin/git/debug-terminal-1.3
      17
assets❯ ag --less --html '(padding|margin).*:.* 20px' | wc -l                                                                                                                                                                                    origin/git/debug-terminal-1.3
      30
assets❯ ag --less --html '(padding|margin).*:.* 30px' | wc -l                                                                                                                                                                                    origin/git/debug-terminal-1.3
      13
assets❯ ag --less --html '(padding|margin).*:.* 2px' | wc -l                                                                                                                                                                                     origin/git/debug-terminal-1.3
       8
assets❯ ag --less --html '(padding|margin).*:.* 3px' | wc -l                                                                                                                                                                                     origin/git/debug-terminal-1.3
      13
assets❯ ag --less --html '(padding|margin).*:.* 8px' | wc -l                                                                                                                                                                                     origin/git/debug-terminal-1.3
       4
assets❯ ag --less --html '(padding|margin).*:.* 12px' | wc -l                                                                                                                                                                                    origin/git/debug-terminal-1.3
       4
```